### ISSUE TITLE
TechRepublic

### ISSUE THREAD

**Issue by barryrowlingson** [matloff/R-vs.-Python-for-Data-Science#15]
A "journalist" on TechRepublic has basically copypastad your article and added a liberal sprinking of "wrote Matloff" to make this mess:

https://www.techrepublic.com/article/r-vs-python-which-is-a-better-programming-language-for-data-science/

enjoy....

**Comment by matloff** [comment]
Yes, thanks, others mentioned it too. A bit lazy on the journalist's part... :-)
